internal/http/handler/session/register.go

```diff
@@ -1,26 +1,26 @@
-package RouteHandler
+package session
 
 import (
 	"encoding/json"
 	"log"
 	"net/http"
 	"sync"
 
-	"github.com/r-erema/wapi/src/Service/Auth"
-	"github.com/r-erema/wapi/src/Service/MessageListener"
-	"github.com/r-erema/wapi/src/Service/SessionWorks"
+	session2 "github.com/r-erema/wapi/internal/repository/session"
+	"github.com/r-erema/wapi/internal/service/auth"
+	"github.com/r-erema/wapi/internal/service/listener"
 )
 
 type RegisterSessionHandler struct {
-	auth         Auth.Interface
-	listener     MessageListener.Interface
-	sessionWorks SessionWorks.Interface
+	auth         auth.Authorizer
+	listener     listener.Listener
```

Comment:
Think about the name of the interface listener.Listener. If it possible to rename and make it more readable?